app.rb

```diff
@@ -38,62 +38,66 @@ def list_of_people
     end
   end
 
-  def create_student_helper(name, age)
-    puts 'Please Enter Student\'s Classroom:'
+  def create_student(name, age)
+    print 'Please Enter Student\'s Classroom: '
     classroom = gets.chomp.to_i
 
-    puts 'Has parent permission? Yes or No:'
+    print 'Has parent permission? [Y/N]: '
     parent_permission = gets.chomp.to_s
 
     case parent_permission
-    when 'Yes'
+    when 'Y'
       # now create the student
       @people << Student.new(age, classroom, name, 'true')
       puts 'Student created successfully!'
-    when 'No'
+    when 'N'
       # now create the student
       @people << Student.new(age, classroom, name, 'false')
       puts 'Student created successfully!'
     else
-      puts 'Invalid Input! Should be Yes or No'
+      puts 'Invalid Input! Should be Y for yes or N for No'
     end
   end
 
+  def create_teacher(age, specialization, name, permission)
+    @people << Teacher.new(age, specialization, name, permission)
+    puts 'Teacher created successfully'
+  end
+
   # create a person
   def create_person
     puts 'Do you want to create: \n 1) a student or \n 2) a Teacher. \n Please choose 1 or 2.'
     choice = gets.chomp.to_i
 
-    puts 'Please Enter Name:'
+    print 'Please Enter Name: '
     name = gets.chomp.to_s
 
-    puts 'Please Enter Age:'
+    print 'Please Enter Age: '
     age = gets.chomp.to_i
 
     case choice
     # when person being created student
     when 1
-      create_student_helper(name, age)
+      create_student(name, age)
 
     # when person being created is a teacher
     when 2
-      puts 'Teacher\'s specialization:'
+      print 'Teacher\'s specialization: '
       specialization = gets.chomp.to_s
 
       # create a teacher
-      @people << Teacher.new(age, specialization, name, 'true')
-      puts 'Teacher created successfully'
+      create_teacher(age, specialization, name, 'true')
     else
       puts 'Invalid Input! Should be 1 or 2'
     end
   end
 
   # create a book
   def create_book
-    puts 'Please Enter Book Title:'
```

Comment:
- [ ] it would be better for all the methods that ask the user for validation and inputs to separate that code that asks for validation and inputs into private methods and then use these  methods in these methods like creating student, teacher ....etc  that way you will achieve the single responsibility principle